### PR TITLE
Adding random letters

### PR DESCRIPTION
**Pull request by yorugac**
Letters are stored as sprites right now, chosen at random instead of lights.

### CODE REVIEW

File: VoltAir/qml/actors/Orb.qml [hunk 1]
```diff
@@ -34,19 +34,22 @@ Actor {
 
     z: 4
 
-    AnimatedImageRenderer {
+    ImageRenderer {
+        function chooseLetter() {
```
Comment (jo-elimu) [inline VoltAir/qml/actors/Orb.qml:38]: @diegowald This part will eventually be connected with the issue you are working on (#31), where a dynamic list of letters are imported and stored. Then that list will be randomly used here.

File: VoltAir/qml/actors/Orb.qml [hunk 2]
```diff
@@ -34,19 +34,22 @@ Actor {
...
+            return Math.random() > 0.5 ? "A" : "B";
+        }
         id: graphic
 
         sizeScale: 1.25
-        rotation: Math.random() * 360.0
-        useGameTime: false
+        // rotation: Math.random() * 360.0
```
Comment (jo-elimu) [inline VoltAir/qml/actors/Orb.qml:44]: @yorugac We don't want the letters to be rotating. Does that mean that we will not be able to create a rotating effect?

File: VoltAir/qml/actors/Orb.qml (hunk 1 above)
Comment (diegowald) [reply VoltAir/qml/actors/Orb.qml:38]: Got it!

File: VoltAir/qml/actors/Orb.qml (hunk 2 above)
Comment (yorugac) [reply VoltAir/qml/actors/Orb.qml:44]: If lights and letters are to be used together and animated, they would have to be separate objects: one rotating and one not.

Comment (jo-elimu) [reply VoltAir/qml/actors/Orb.qml:44]: @yorugac Ok, that sounds like the best solution.

**Pull request merged**